### ISSUE TITLE
I get Excel xlsx file not supported

### ISSUE THREAD

**Issue by nevasha27** [toransahu/excel2json-3#5]
excel2json-3 --file './Assessment.xlsx' 
Excel xlsx file; not supported
'NoneType' object is not iterable

here what pip list shows
Package          Version
---------------- ----------
appdirs          1.4.4
certifi          2020.12.5
chardet          4.0.0
distlib          0.3.1
et-xmlfile       1.1.0
excel2json       1.0.1
excel2json-3     0.1.6
filelock         3.0.12
idna             2.10
openpyxl         3.0.7
pip              21.1.1
pipenv           2020.11.15
requests         2.25.1
setuptools       49.2.1
six              1.15.0
urllib3          1.26.4
virtualenv       20.4.4
virtualenv-clone 0.5.4
wheel            0.36.2
xlrd             2.0.1

What could I be doing wrong?

I have: Python 3.9.4

I followed the instructions in USES.md

>python3 
Python 3.9.4 (v3.9.4:1f2e3088f3, Apr  4 2021, 12:32:44) 
[Clang 6.0 (clang-600.0.57)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> from excel2json import convert_from_file
>>> EXCEL_FILE = './Assessment.xlsx'
>>> convert(EXCEL_FILE)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
NameError: name 'convert' is not defined

**Comment by nevasha27** [comment]
I am a newbee to python and so may well have questionable install when it comes to excel2json.

**Comment by GamerBoy2K** [comment]
Facing the same problem
<samp>
Excel xlsx file; not supported
'NoneType' object is not iterable
</samp>

**Comment by DLTKkrystlepalmario** [comment]
Is there an update regarding this issue?

**Comment by hoferdan** [comment]
Even though this is an old issue, i figured out the problem.
`excel2json-3` is using the package `xlrd` which stopped supporting xlsx files in version 2.0.0.

**Comment by dsdhaiti** [comment]
Are there any other packages you can suggest that does the same job?

**Comment by yilverdeja** [comment]
> Even though this is an old issue, i figured out the problem. `excel2json-3` is using the package `xlrd` which stopped supporting xlsx files in version 2.0.0.

Good catch, it worked for me when I downgraded the `xlrd` version to 1.2.0 using

```
pip install xlrd==1.2.0
```
